src/Fields/mapreduce.jl

```diff
@@ -135,3 +135,6 @@ function Base.isapprox(
     d = norm(x .- y)
     return isfinite(d) && d <= max(atol, rtol * max(norm(x), norm(y)))
 end
+
+Base.:(==)(field1::Field, field2::Field) =
```

Comment:
is this used anywhere performance critical? or is it mostly for convenience?

Reply:
Convenience. I used it in a Jacobian unit test, and it's also helpful for debugging.